src/web/components/elements/InstagramBlockComponent.tsx

```diff
@@ -1,24 +1,29 @@
 import React from 'react';
 
-import { unescapeData } from '@root/src/lib/escapeData';
 import { css } from 'emotion';
 
-const widthOverride = css`
-	iframe {
-		/* The instagram embed js hijacks the iframe and calculated an incorrect width, which pushed the body out */
-		min-width: 300px !important;
-	}
+const fullWidthStyles = css`
+	width: 100%;
 `;
 
+/**
+ * Note the iframe created in this compoenet is dependent on the embedIFrame function being
+ * called to be size correctly.
+ * src/web/browser/embedIframe/embedIframe.ts
+ */
 export const InstagramBlockComponent: React.FC<{
 	element: InstagramBlockElement;
-}> = ({ element }) => {
+	index: number;
+}> = ({ element, index }) => {
 	return (
-		<div className={widthOverride}>
-			<div
-				data-cy="instagram-embed"
-				dangerouslySetInnerHTML={{ __html: unescapeData(element.html) }}
-			/>
-		</div>
+		<iframe
+			className={`${fullWidthStyles} js-embed__iframe`}
```

Comment:
I don't think we need this class `js-embed__iframe`

Reply:
Well... In the case where the click to view is not in operation eg if we just roll out this PR without the ClickToView changes, this class is required as it gets picked up by this piece of code:

https://github.com/guardian/dotcom-rendering/blob/main/src/web/browser/embedIframe/embedIframe.ts#L3-L4

This is how the UnsafeEmbedBlockComponent work which Im basing this off. There is most probably a better way of doing this.

Reply:
😱 that's the first time I've seen a classname used as a selector in DCR! But there it is.